packages/theme/src/lib/legacy-theme.ts

```diff
@@ -393,8 +398,9 @@ export const legacyTheme = {
       styleOverrides: {
         root: {
           boxShadow: 'none',
-          fontWeight: tokens.fontWeightsBold,
           textTransform: 'none',
+          padding: '6px 12px',
+          lineHeight: tokens.lineHeightsBody1,
```

Comment:
I would remove these hardcoded padding and line heights for button. I added the values for the different sizes in my iconbutton ticket and also set the defaultprop for size. https://github.com/Availity/element/blob/43c424e2f33b0dcd9dca51ec253e54ea3309630b/packages/theme/src/lib/legacy-theme.ts#L423

Reply:
Makes sense. Removed.